tutorials/plugins/gdnative/gdnative-cpp-example.rst

```diff
@@ -388,6 +485,348 @@ We're finally ready to run the project:
 
 .. image:: img/gdnative_cpp_animated.gif
 
+Adding properties
+-----------------
+GDScript allows you to add properties to your script using the export keyword.
```

Comment:
Use code markup for export,
<pre>``export``</pre>

Reply:
I thought I was being very consistent with that, missed one :)